lookup/simple_env.go

```diff
@@ -3,14 +3,12 @@ package lookup
 import (
 	"fmt"
 	"os"
-
-	"github.com/docker/libcompose/project"
 )
 
 type OsEnvLookup struct {
 }
 
-func (o *OsEnvLookup) Lookup(key, serviceName string, config *project.ServiceConfig) []string {
+func (o *OsEnvLookup) Lookup(key string) []string {
```

Comment:
@vdemeester I've embedded libcompose in other projects and the other values are quite useful.  I know the default impl doesn't use it but it is nice if you want to do context based lookup of env vars.  Also, just a side note, `ServiceConfig` doesn't actually include the service name in it so a lot of times we pass name+config to give full context.